apps/src/templates/progress/ProgressButton.jsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * New style button. Eventually we'd like to have this used more broadly (at which
+ * point it should be renamed to something more generic)
+ * This particular button is designed to operate in contexts where we have a solid
+ * background. When we're a button on top of an image, we may want something different.
+ */
```

Comment:
I think putting it here for now is the right call, as long as we have the organizational memory to move it when we're ready to use it elsewhere :grin:.